app/src/main/java/com/dessalines/thumbkey/keyboards/ESMessageEase.kt

```diff
@@ -618,13 +642,358 @@ val KB_ES_MESSAGEEASE_SHIFTED =
         ),
     )
 
+val KB_ES_MESSAGEEASE_NUM =
```

Comment:
Is this one really necessary? I'm looking at our global numeric keyboard, and it looks nearly identical. 

Also does messageease have different numeric layouts for each language? If not then this should pry go into its own file.

Reply:
I'd have to check, but I'm fairly sure that ME had slight differences between languages. At the very least you'll want a "global MessagEase numeric keyboard"

Reply:
That's fine, someone else also has the same idea: 

https://github.com/dessalines/thumb-key/pull/687

As long as it lives in its own layout file, and is used by every messageease keyboard.

Reply:
I'll check the english layout in ME tomorrow and if the num layer is the same, split it into a separate keyboard in my PR